aries-site/src/components/accessibility/Video.js

```diff
@@ -0,0 +1,72 @@
+import React, { useContext, useState } from 'react';
+
+import { Accordion, AccordionPanel, Anchor, AnnounceContext, Box } from 'grommet';
+import { set } from 'react-ga';
+import { FormDown, FormUp } from 'grommet-icons';
+
+const desc = `The video shows Bill, a white man with blondish brown hair 
+  and a mustache, in his office wearing a long-sleeved lavender-blue 
+  dress shirt with the HPE rectangular-shaped, multi-color logo
+  against a black background as the background.`
+
+export const AccessVideo = () => {
+  const announce = useContext(AnnounceContext);
+
+  const [expanded, setExpanded] = useState(false);
+
+  return (
+    <>
+      <Box 
+        gap="small" 
+        width="large" 
+        pad={{ bottom: 'large' }}
+      >
+        <Box 
+          a11yTitle="Audio Desciption"
+          aria-description="Click to show audio description"
+          gap="small" 
+          direction="row" 
+          onClick={() => {
+            setExpanded(!expanded);
+            expanded ? announce("Hid audio description") : 
+              announce("Opened audio description");
+          }} 
+        >
+          Audio Description
+          {
+            expanded ? 
+              <FormUp a11yTitle="Up arrow" /> : 
+              <FormDown a11yTitle="Down arrow" />
+          }
+        </Box>
+        {expanded && desc}
```

Comment:
This pattern of having the audio description visibly ahead of the video is new to me. Can you share any resources, examples, etc. which are inspiring this direction?

Reply:
sure! after working with bill with numerous testing rounds, he had shared that the audio description through `aria-description` and `aria-describedby` was not consistent in the reading of the description. while when testing with VO i was able to hear the description at the video start and end, he was not able to get it read consistently with JAWS or NVDA.

to ensure that the description would be read aloud at some point, bill mentioned that another pattern he has seen before is to provide a way on the same page as the video for screen readers to get the description, either as hidden text on the page or a button that triggered the description to be shown.

in response to why `aria-description` and `aria-describedby` were flawed, it seems that the `<video />` doesn't have great screen reader support with related props as shown [here](https://www.powermapper.com/tests/screen-readers/media/video-aria-labelledby/) and [here](https://www.powermapper.com/tests/screen-readers/media/video-aria-label/). `aria-describedby` also seems to be [mostly used with form elements,](https://www.powermapper.com/tests/screen-readers/labelling/img-aria-describedby/) but when used with other elements like `<img />`, the support changes drastically as shown [here](https://www.powermapper.com/tests/screen-readers/labelling/img-aria-describedby/). the same could be said for `aria-decription` since it functions the same way.

based on my testing with VO and with bill with JAWS and NVDA, it seems there is varied support around both of these aria attributes and so we took the approach of displaying the audio description in some way on the screen so users with screen readers can find it. we also included the `a11yTitle` for the audio transcript file to relay that an audio description could also be found there, which is [common practice](https://www.3playmedia.com/learn/popular-topics/audio-description/#formats).

Reply:
Feels like we should have guidance on DS site of how to implement this. I don't know if we have a "Video" component page, but this kind of information would be helpful and important to share there -- even if videos are less common UI elements for our apps.

Reply:
Couple of questions on the approach though:
- I'm wondering if the audio description should go beneath the video instead? It feels a bit too prominent. Also it feels more like a "Video Description" to me than an audio description, because we are describing the visuals.
- Could we potentially mirror the design approach that Ted Talks takes (showing the transcript inline on the page) -- maybe for our use we can mirror the "Show Code" approach but instead have "Read Transcript" as the collapsible label: Example on TedX approach when you click "Read Transcript" [https://www.ted.com/talks/jill_bolte_taylor_my_stroke_of_insight](https://www.ted.com/talks/jill_bolte_taylor_my_stroke_of_insight)
- The "Audio Description" is already repeated in the transcript. Is there any reason we need to have both?

Reply:
Agreed with having guidance reflecting these Video considerations on the DS. To your other points:

- the terms "audio description" and "video description" are used interchangeably in the a11y community and usually come before the video to properly set up the context of the video. i had it as "audio description" because we are describing the video through audio for screen reader use and bill had mentioned that was the terminology he was used to. i can change to "video description" if you feel that's more appropriate!
- i had originally thought about the "show code" approach and making use of that component/pattern as it exists in the DS, but my big thing there was the accessibility of the code snippet piece as logged in [this issue](https://github.com/grommet/hpe-design-system/issues/1821). the other piece to this was the fact that transcripts should allow for a user to download the file, as pointed out by bill. i think it could be worth it to explore how we could add the inline transcript and allow for downloading the file in our Video component. i could approach this in the same way that i'm handling the expandable description if that would be better but bill noted his experience currently is accessible and up-to-standard.
- the "audio description" is included in the transcript file as it should be accessible at the page level that the video is on and included in the file itself.

let me know your thoughts!

Reply:
I am very much in support of ensuring all the necessary accessibility requirements are met so those using assistive technologies can have an equal experience to those not. That being said, I want to be mindful that the assistive technology support doesn't clutter the UI or disrupt the usability of a page for users not relying on the assistive technologies, as this could create a poor experience in the opposite direction. That's where some of my design comments are coming from.

Personally, I found the "Audio Description" button to be a bit disruptive to the page flow because it comes first and draws a lot of attention to itself, which is why I'd be more in favor of it coming below the video. Good note about code snippet issue, I'm fine with exploring it implemented how you handled it as opposed to the show code approach but maybe placing underneath the video?

For the case of needing to be able to download the transcript, maybe we can have the transcript appear inline but also provide a download button for those who would want to download it? I think an icon-only button would be sufficient with an `a11yTitle="Download transcript"` ?